docs/Build.md

```diff
@@ -0,0 +1,68 @@
+# Build Documentation
+
+## Build Information
+
+The build process is performed using the `build.ps1` PowerShell script located in the root of the project.
+The build script combines the module code with the generated MAML documentation.
+The only prerequisites for building this project are PowerShell and the [PlatyPS][PlatyPS] module.
+
+The build process will create the output in the `bin` directory beneath the project root.
+
+[PlatyPS]: https://github.com/PowerShell/platyPS
+
+## Building the Module
+
+To build the project from PowerShell:
+
+```powershell
+Install-Module -Scope CurrentUser PlatyPS
+git clone https://github.com/PowerShell/WindowsPowerShellCompatibilityPack.git
```

Comment:
We need this updated now that the repo is renamed, same for other references to the old name